src/HttpAdapterFactory.php

```diff
@@ -47,6 +47,20 @@ class HttpAdapterFactory
         self::ZEND2             => 'Ivory\HttpAdapter\Zend2HttpAdapter',
     );
 
+    private static $guessMap = array(
+        self::GUZZLE_HTTP => '\GuzzleHttp\Client',
+        self::GUZZLE      => '\Guzzle\Http\Client',
+        self::BUZZ        => '\Buzz\Browser',
+        self::ZEND2       => '\Zend\Http\Client',
+        self::ZEND1       => '\Zend_Http_Client',
+        self::CAKE        => '\HttpSocket',
+        self::HTTPFUL     => '\Httpful\Request',
+        self::REACT       => '\React\HttpClient\Request',
+        self::CURL        => 'curl_exec',
```

Comment:
Can you rely on `curl_init` here in order to be consistent with other part of the code?